rotkehlchen/tests/unit/decoders/test_curve.py

```diff
@@ -1461,3 +1461,64 @@ def test_3pool_remove_liquidity(ethereum_transaction_decoder, ethereum_accounts)
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0x60b0f1919cf4ee46d1A8D63428276512814de570']])
+def test_remove_from_aave_pool(ethereum_transaction_decoder, ethereum_accounts):
+    """
+    Test that if liquidity is removed from a pool with a(aave) tokens,
```

Comment:
Please fix this typo in a subsequent PR

```suggestion
    Test that if liquidity is removed from a pool with aave atokens,
```